src/routes.js

```diff
@@ -95,15 +69,16 @@ export default function getRoutes(
 			request.log(['info'], chalk.green(`Request received for ${request.url.href} (${requestLanguage})`));
 
 			const render$ = request.authorize()  // `authorize()` method is supplied by anonAuthPlugin
-				.flatMap(renderRequestMap[requestLanguage]);
+				.flatMap(renderRequestMap[requestLanguage])
+				.do(() => request.log(['info'], chalk.green('HTML response ready')));
 
 			render$.subscribe(
 				({ result, statusCode }) => {
 					// response is sent when this function returns (`nextTick`)
-					const response = reply(result).code(statusCode);
-					initTrackingCookie(request, response);
+					const response = reply(result)
+						.code(statusCode);
+					reply.track(response, 'session');
 
-					request.log(['info'], chalk.green('HTML response ready'));
 					if (reply.request.app.setCookies) {
```

Comment:
this handling will almost certainly be moved in with other tracking/cookie setting code in https://meetup.atlassian.net/browse/MW-331